extensions/cornerstone-dicom-sr/src/viewports/OHIFCornerstoneSRViewport.tsx

```diff
@@ -419,6 +421,9 @@ OHIFCornerstoneSRViewport.propTypes = {
   dataSource: PropTypes.object,
   children: PropTypes.node,
   customProps: PropTypes.object,
+  viewportOptions: PropTypes.object,
```

Comment:
Is it feasible to define a `ViewportOptions` type now?

Reply:
Not really - this happens to embed a cornerstone viewport, so we can probably get away with saying it is a publicviewportoptions instance.